<?xml version="1.0"?>
<launch>
    <!--include other launch files-->
    <include file="$(find nao_bringup)/launch/nao_full_py.launch">
    </include>

    <include file="$(find nao_apps)/launch/tactile.launch">
    </include>

    <!--launch data_acquisition node-->
    <node name="DataAcquisitionNode" pkg="tutorial_4" type="data_acquisition_node.py"
	output="screen" respawn="true"/>

</launch>
